### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by divyansh10108**
## My first successful PR
- [x] added my card 
- [x] Created my first contribution
- [x] Learned about PR's and basic html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,30 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *divyansh10108* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Divyansh</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/Divyans01671370" target="_blank">catman</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://twitter.com/Divyans01671370" target="_blank">@catman</a>
```

File: index.html [hunk 2]
```diff
@@ -115,6 +115,30 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">Hey, I am Divyansh Gaba. My first contribution to github. Looking forward to learn more about such technologies</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.theodinproject.com/" target="_blank" title="First resource">TOP</a>
```
Comment (TenTraicion) [inline index.html:130]:
```suggestion
                <a href="https://www.theodinproject.com/" target="_blank" title="First resource">The Odin Project</a>
```

File: index.html [hunk 3]
```diff
@@ -115,6 +115,30 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="Second resource"> FreeCodeCamp</a>
+              </li>
+              <li>
+                <a href="https://devdocs.io/" target="_blank" title="Third resource">Documentation for everything</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*khaja_shaik* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:141]:
```suggestion
        <!-- ________*divyansh10108* Contributor card END ________  -->
        
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindly make the requested changes and commit. Thanks!

**Comment by CandidDeer** [comment]
Thank you for resolvind the requested changes. Your card has been added to the live website and will be visible in a couple of mins. Well done!

**Pull request merged**